tests/test_base_template.py

```diff
@@ -122,9 +122,10 @@ def test_extracting_placeholders(template_content, template_subject, expected):
     "content,prefix, expected_length, expected_replaced_length",
     [
         ("The quick brown fox jumped over the lazy dog", None, 44, 44),
-        ("深", None, 3, 3),
+        # should be replaced with a ?
+        ("深", None, 1, 1),
         ("'First line.\n", None, 12, 12),
-        ("\t\n\r", None, 0, 0),
+        (" \n\r", None, 0, 0),
```

Comment:
Shouldn’t need to change this test – tab should still be replaced with space and then count as `0` length.